### ISSUE TITLE
Should EUR currency symbol come before or after value?

### ISSUE THREAD

**Issue by bjanssen-pon** [reactioncommerce/reaction#6517]
For the EURO currency there are two types of representation:

- 1234 EUR 
- € 1234

In the definitions of the currency, I have the format `%v %s`. The format should be `%s %v`.

Location: `api-utils/CurrencyDefinitions.js`

**Comment by aldeed** [comment]
@bjanssen-pon Can you link to any international standard you are following? I think whether € is left or right varies both by region and by language, so if we change it one way, someone else will complain that it's wrong for them. Unfortunately, Reaction does not use currency formatting that can be different per region or per language.

If there is a good international standard to follow, that makes sense. Or if someone has statistics showing that one way is far more common. The only standard I have found is [EU style guide](http://publications.europa.eu/code/en/en-370303.htm) but that shows both ways depending on language.

**Comment by HarisSpahija** [comment]
> Position of the euro sign (€) in amounts
> The euro sign is followed by the amount without space:
> 
> a sum of €30
> NB:
> The same rule applies in Dutch, Irish and Maltese. In all other official EU languages the order is reversed; the amount is followed by a hard space and the euro sign:
> une somme de 30 €

I think it is good to follow this standard. 
Wikipedia has a nice table of what countries use the euro sign before and after.

https://en.wikipedia.org/wiki/Language_and_the_euro#Summary

**Comment by aldeed** [comment]
I agree in theory but to support language-specific formatting would take some work. I created a new issue that outlines a possible approach, but I may have forgotten something.

https://github.com/reactioncommerce/reaction/issues/6511

I'll leave this issue open as a discussion of which EUR symbol position should be the default when language isn't known.

**Comment by bjanssen-pon** [comment]
Thanks for the discussion!

As for all companies of Pon, we use the international notation for the EUR currency where the euro symbol is in front of the amount.
As for this, i use ISO-4217.

**Comment by bjanssen-pon** [comment]
Alsof, there van be two types of notations... 
a. € 15,00
b. 15,00 EUR

**Comment by aldeed** [comment]
From ISO-4217 Wikipedia page:

<img width="640" alt="Screenshot 2020-05-21 12 23 25" src="https://user-images.githubusercontent.com/3012067/82586931-f604e080-9b5d-11ea-8983-3e86311c261e.png">

**Comment by bjanssen-pon** [comment]
Yes, correct. That is the option b i posted.
But i want to use the symbol notation.
Is it an option to override the default somehow from the admin? Would make sense. 
I could override it in versions 2.x by adjusting the currency settings file.

**Comment by aldeed** [comment]
Overriding is no longer possible without code changes. That is something we lost, but could be added back in the future.

**Comment by bjanssen-pon** [comment]
It would be very nice to have that option again or have the option to have multiple representations and select it in the admin.
